### ISSUE TITLE
[Forms] Enhance question input to match the design

### ISSUE THREAD

**Issue by aniri** [code4romania/monitorizare-vot-ong#280]
Go to forms page, add a section and a question.

Merge the question text input and question type drowdown as seen in the design.

![95456418-47f7a880-093d-11eb-8705-d56988f528eb](https://user-images.githubusercontent.com/1358055/95460902-34206680-097e-11eb-9507-b9d4485c9228.png)

Please see complete figma here: https://www.figma.com/file/61NHR7fwmV1Dsv4Pb6F8T7/code4ro_mv_user-management_2018.04.21-(Copy)?node-id=754%3A0 , check under 'create form'

**Comment by Botosio** [comment]
Questions, so we would need to save 2 values for this correct?
The question text and the dropdown value.  Merge the UI into one should be that hard (custom css, I don't believe bootstrap lets you do this).  

I noticed when you make changes to the dropdown the "options" do not reflect the change, do you want that happen in this change also?

**Comment by aniri** [comment]
@Botosio yes, we need to save 2 values for this, as they are saved now. We just need to merge them into one visually. 

I don't think there's a need for updating options when the dropdown changes. The question type is used when displaying the forms to the end users, if they can pick one of more options.

**Comment by Botosio** [comment]
> @Botosio yes, we need to save 2 values for this, as they are saved now. We just need to merge them into one visually.
> 
> I don't think there's a need for updating options when the dropdown changes. The question type is used when displaying the forms to the end users, if they can pick one of more options.

So if they choose multiple-choice, add 3 options. then go back and make it a single choice. The form still shows 3 options.
So at that time, should we make it so they could only have 1 choice?  Up to you, let me know.

**Comment by aniri** [comment]
@Botosio I think the naming of the dropdown values is a bit confusing :smiley: The forms configured here will be answered by the users on the mobile apps.

So, if a question has 3 options and is of type 'single choice' when the user answers the question, they will only be allowed to pick one of the 3 options. If the type is 'multiple choice' they can pick more than one. So the number of available options is not affected by the question type, just the allowed user behaviour.

I hope this makes sense now :D

**Comment by Botosio** [comment]
Ohhhh ok that I'm with you now!
